Procfile.dev

```diff
@@ -1,2 +1,3 @@
 web: bundle exec puma -C config/puma.rb -e ${RACK_ENV:-development}
-worker: bundle exec sidekiq -C config/sidekiq.yml -e ${RACK_ENV:-development}
\ No newline at end of file
+worker: bundle exec sidekiq -C config/sidekiq.yml -e ${RACK_ENV:-development}
+webpacker: ./bin/webpack-dev-server
```

Comment:
Do we need to add this to the production procfile?

Do you know why we need to specify this when in the past we didn't?

Reply:
Not sure if we need to add to the production Procfile. Webpacker detects changes to the javascript code so you don't need to refresh the page: https://webpack.js.org/concepts/hot-module-replacement/

However, since the assets are compiled during build time (and assets shouldn't change) I don't believe we're supposed to include webpacker. I'm not 100% confident though, so I'll test against staging-2.